client/packages/electron/src/keyboardScanner/keyboardScanner.ts

```diff
@@ -0,0 +1,231 @@
+import { BrowserWindow } from 'electron';
+import { IPC_MESSAGES } from '../shared';
+import { BarcodeScanner } from '@common/hooks';
+
+// Some modifier keys are passed through (no preventDefault)
+const keysToPassThrough = [
+  'Backspace',
+  'Tab',
+  'Enter',
+  'Shift',
+  'Control',
+  'Alt',
+  'Pause',
+  'CapsLock',
+  'Escape',
+  'Space',
+  'PageUp',
+  'PageDown',
+  'End',
+  'Home',
+  'ArrowLeft',
+  'ArrowUp',
+  'ArrowRight',
+  'ArrowDown',
+  'PrintScreen',
+  'Insert',
+  'Delete',
+  'Clear',
+];
+
+// Scan Log
+const PROCESSOR_INTERVAL = 50;
+
+type KeyEvent = { input: Electron.Input; ms: number };
+
+// Some keys need to be removed (mention about the thig)
```

Comment:
Unsure what the last part means sorry: `(mention about the thig)`

Reply:
Oops, lol the thig was removed: fe936ba938a244f34dcdc05fb744b020bbf8d305